inventory/host_vars/www.openfoodnetwork.org.za/config.yml

```diff
@@ -0,0 +1,12 @@
+---
+domain: www.openfoodnetwork.org.za
+rails_env: production
+
+mail_domain: openfoodnetwork.org.za
+
+certbot_domains:
+  - www.openfoodnetwork.org.za
+  - openfoodnetwork.org.za
+
+unicorn_workers: 2
+swapfile_size: 1G
```

Comment:
I'm wondering if we should make this the standard now.

Reply:
I went for some conservative options as I'm not sure about their specs, but yes, we can probably set some bigger defaults soon for all servers.